src/main/java/mate.academy/Main.java

```diff
@@ -3,15 +3,34 @@
 import java.util.List;
 import mate.academy.lib.Injector;
 import mate.academy.model.Product;
+import mate.academy.service.FileReaderService;
+import mate.academy.service.ProductParser;
 import mate.academy.service.ProductService;
 
 public class Main {
 
     public static void main(String[] args) {
-        // Please test your Injector here. Feel free to push this class as a part of your solution
+        // Create an instance of the Injector
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+
+        FileReaderService fileReaderService = injector.getInstance(FileReaderService.class);
+        ProductParser productParser = injector.getInstance(ProductParser.class);
+        final ProductService productService = injector.getInstance(ProductService.class);
+
+        // Test the FileReaderService functionality
```

Comment:
Thanks for comments,
but code should be readable without them
For the future, remove comments